README.md

```diff
@@ -28,7 +28,7 @@ Set up [Vundle](https://github.com/VundleVim/Vundle.vim) then add `Plugin
 
 Set up [vim-plug](https://github.com/junegunn/vim-plug). In your .vimrc, between
 the lines for `call plug#begin()` and `call plug#end()`, add the line `Plug
-'cespare/vim-toml'`. Then run `:PlugInstall` from a fresh vim instance.
+'cespare/vim-toml'`. Save `:w` and source `so%` your .vimrc and then then run `:PlugInstall`.
```

Comment:
Double "then".

But how about something like

> Reload your .vimrc and then run `:PlugInstall`.

following the vim-plug readme example? I don't think we need to explain how to reload the vimrc.